Formula/open-babel.rb

```diff
@@ -1,26 +1,11 @@
 class OpenBabel < Formula
   desc "Chemical toolbox"
-  homepage "http://www.openbabel.org"
-
+  homepage "https://openbabel.org"
+  url "https://downloads.sourceforge.net/project/openbabel/openbabel/2.4.0/openbabel-openbabel-2-4-0.tar.gz"
```

Comment:
Would it be worth switching to the GitHub release download url instead?
https://github.com/openbabel/openbabel/releases/tag/openbabel-2-4-0
I think files are now just mirrored across from GitHub to sourceforge.